src/puppy/platforms/win32/platform.nim

```diff
@@ -55,26 +48,21 @@ proc fetch*(req: Request): Response {.raises: [PuppyError].} =
       raise newException(
         PuppyError, "WinHttpConnect error: " & $GetLastError()
       )
-
     var openRequestFlags: DWORD
     if req.url.scheme == "https":
       openRequestFlags = openRequestFlags or WINHTTP_FLAG_SECURE
-
     var objectName = req.url.path
     if req.url.search != "":
       objectName &= "?" & req.url.search
-
```

Comment:
Same idea here, please don't change file formatting that isn't necessary to change.